### PR TITLE
feat: claim issue date, deemed date of service, response deadline (CMC-510)

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link (if applicable) ###
[CMC-510](url)

### Change description ###
This PR implements the calculation of issue date, deemed date of service, response deadline and confirmation of service dates.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/CaseField.json [hunk 1]
```diff
@@ -89,6 +89,22 @@
     "FieldType": "Date",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "claimSubmittedDateTime",
```
Comment (swalker125) [inline ccd-definition/CaseField.json:95]: why is this field needed as well as claimIssueDate? Are they not the same?

**Review by swalker125 — COMMENTED** [review]
will look through the rest later today, just a question around what claimSubmittedDateTime is for

File: ccd-definition/CaseField.json (hunk 1 above)
Comment (sabahirfan) [reply ccd-definition/CaseField.json:95]: Nope these are two different dates. `claimSubmittedDateTime` is the timestamp when claim is submitted and required on the sealed claim pdf, while `claimIssueDate ` can be next working day and after 4pm etc.

Comment (swalker125) [reply ccd-definition/CaseField.json:95]: are you sure? claimIssuedDate is just when I submit the form right? I don't see why that needs to be next working day etc. In that case they are the same. If not the same, why do we need to store this at all? Is it not throw away info?

**Review by dharmendrak — APPROVED** [review]

File: ccd-definition/CaseField.json (hunk 1 above)
Comment (sabahirfan) [reply ccd-definition/CaseField.json:95]: yes, I am sure. look at the story for requirements for this.
```
IssueDate =
now
+ 1 day if >= 16:00 PM
+ 1 day till next working day
```

Comment (swalker125) [reply ccd-definition/CaseField.json:95]: then claimIssueDate can probably be removed and we will just use claimIssueDateAndTime? I don't see why we need both fields

Comment (sabahirfan) [reply ccd-definition/CaseField.json:95]: As mentioned earlier both dates are needed on sealed claim PDF.

**Review by karthikkannan-lemuria — APPROVED** [review]

**Pull request merged**